tests/testthat/test-hover_reload_button.R

```diff
@@ -0,0 +1,33 @@
+context("Reload functionality")
+
+test_that("Test for three components in the button. Type is a 'button'.",
```

Comment:
Let's reword to:
> "hover_reload_button has expected structure"

FYI, I think the builds are failing because test_that is expecting a single vector for the description, e.g. the `desc` param.